docs/pages/management/export-audit-events/splunk.mdx

```diff
@@ -0,0 +1,791 @@
+
+```code
+$ sudo systemctl restart SplunkForwarder
+```
+
+## Step 3/4. Run the Event Handler plugin
+
+Now that you have configured your Universal Forwarder to receive logs via HTTP
+and forward them to Splunk, you will ensure that the Teleport Event Handler
+plugin is configured to authenticate to the Universal Forwarder and your
+Teleport cluster, then run the Event Handler.
+
+### Complete the Event Handler configuration
+
+Earlier, we generated a file called `teleport-event-handler.toml` to configure
+the Event Handler plugin. This file includes settings similar to the following:
+
+```toml
+storage = "./storage"
+timeout = "10s"
+batch = 20
+namespace = "default"
+
+[forward.fluentd]
+ca = "/home/ca.crt"
+cert = "/home/client.crt"
+key = "/home/client.key"
+url = "https://localhost:9061/test.log"
+
+[teleport]
+addr = "example.teleport.com:443"
+identity = "identity"
+```
+
+Update the configuration file as follows.
+
+Change `forward.fluentd.url` to the following:
+
+```text
+url = "https://localhost:9061/services/collector/raw?token=MYTOKEN"
+```
+
+Ensure the URL includes the scheme, host and port of your Universal Forwarder's
+HTTP intput, plus the URL path that the Universal Forwarder uses for raw data
+(`/services/collector/raw`). 
+
+Replace `MYTOKEN` with the token you generated earlier for the Splunk Universal
+Forwarder. If you are running the Universal Forwarder and Event Handler on
+separate hosts, replace `localhost` with your Universal Forwarder's IP address
+or domain name.
+
+Change `forward.fluentd.session-url` to the same value as `forward.fluentd.url`,
+but with the query parameter key `&noop=` appended to the end:
+
+```text
+session-url = "https://localhost:9061/services/collector/raw?token=MYTOKEN&noop="
+```
+
+For audit logs related to Teleport sessions, the Event Handler appends routing
+information to the URL that our HTTP input configuration does not use. Adding
+the `noop` query parameter causes the Event Handler to append the routing
+information as the parameter's value so the Universal Forwarder can discard it.
+
+Change `teleport.addr` to the host and port of your Teleport Proxy Service, or
+the Auth Service if you have configured the Event Handler to connect to it
+directly, e.g., `mytenant.teleport.sh:443`.
+
+<Tabs>
+<TabItem label="Connect to the Proxy Service">
+
+Assign `teleport.identity` to a path to the identity file you exported earlier,
+e.g., `/home/auth.pem`.
+
+Ensure that the Event Handler can read these credentials:
+
+```code
+$ chmod +r auth.pem
+```
+
+</TabItem>
+<TabItem label="Connect to the Auth Service">
+
+Assign `teleport.ca`, `teleport.cert`, and `teleport.key` to the paths of the
+TLS credentials you generated earlier. Respectively, these are the certificate
+authority, certificate, and private key.
+
+Ensure that the Event Handler can read these credentials:
+
+```code
+$ chmod +r teleport.ca teleport.cert teleport.key
+```
+
+</TabItem>
+</Tabs>
+
+### Start the Event Handler
```

Comment:
Shouldnt we recommend running the Event Handler as a systemd service so it persists reboots?